systemtests/src/test/java/io/enmasse/systemtest/bases/clients/ClientTestBase.java

```diff
@@ -293,6 +293,7 @@ protected void doDrainQueueTest(AbstractClient sender, AbstractClient receiver)
         arguments.put(ClientArgument.ADDRESS, dest.getSpec().getAddress());
         arguments.put(ClientArgument.COUNT, Integer.toString(expectedMsgCount));
         arguments.put(ClientArgument.MSG_CONTENT, "msg no. %d");
+        arguments.put(ClientArgument.TIMEOUT, "10");  // In seconds, maximum time the consumer waits for a single message
```

Comment:
I think since this is a receiver argument, it would be better moved down a few lines.